doc/development/guidelines.md

```diff
@@ -156,12 +159,18 @@ requires = [
 ]
 build-backend = "setuptools.build_meta"
 
-# allow openalea to be a namespace package
+# src is okay as openalea is a namespace package
```

Comment:
where = ["src"] is OK if the layout is src/openalea/my_package

Reply:
okay I update the comment (see above)